src/Akeneo/Tool/Component/StorageUtils/Indexer/ProductIndexerInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Akeneo\Tool\Component\StorageUtils\Indexer;
+
+/**
+ * Interface ProductIndexerInterface
+ *
+ * @author    Nicolas Marniesse <nicolas.marniesse@akeneo.com>
+ * @copyright 2019 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface ProductIndexerInterface
+{
+    /**
+     * @param string $productIdentifier
+     * @param array  $options
+     */
+    public function indexFromProductIdentifier(string $productIdentifier, array $options = []);
```

Comment:
```suggestion
    public function indexFromProductIdentifier(string $productIdentifier, array $options = []): void;
```

same for other methods

Reply:
Fix in https://github.com/akeneo/pim-community-dev/pull/10646/commits/22f67552d5edc696267d27f2af47774cd7efab17